src/vs/workbench/parts/terminal/electron-browser/terminalPanel.ts

```diff
@@ -27,6 +27,18 @@ import { StandardMouseEvent } from 'vs/base/browser/mouseEvent';
 import { TPromise } from 'vs/base/common/winjs.base';
 import URI from 'vs/base/common/uri';
 
+// List of regular expressions, for matching shells, functions to format a path, and booleans indicating whether the shell uses Windows-style paths
+const WIN_PATH_FORMATTERS: [RegExp, (path: string) => string, boolean][] = [
```

Comment:
An interface would be nicer but it does add some verbosity to `WIN_PATH_FORMATTERS` as the array elements would be switched from arrays to objects. Would doing something like:

 ```typescript
type PathFormatter = [Regexp, (path: string) => string, boolean];

const WIN_PATH_FORMATTERS: PathFormatter[] = [...]
```

be any better? Or given I haven't really seen this used in VS Code would it be better to stick with interfaces?